core/src/rpc.rs

```diff
@@ -738,7 +740,12 @@ impl JsonRpcRequestProcessor {
                         bigtable_blocks.retain(|&slot| slot <= end_slot);
                         bigtable_blocks
                     })
-                    .unwrap_or_else(|_| vec![]));
+                    .map_err(|_| {
```

Comment:
well, I'm actively censoring not to embed arbitrary bigtable error in the http response. :)